huggingface/azureml/hf-ort.py

```diff
@@ -0,0 +1,146 @@
+import os
+import requests
+import sys
+import shlex
+from datetime import datetime
+import argparse
+
+# AzureML libraries
+import azureml.core
+from azureml.core import Experiment, Workspace, Datastore, Run, Environment
+from azureml.core.compute import ComputeTarget, AmlCompute, AksCompute
+from azureml.core.compute_target import ComputeTargetException
+from azureml.core import ScriptRunConfig
+from azureml.core.runconfig import PyTorchConfiguration
+
+# Check core SDK version number
+print("SDK version:", azureml.core.VERSION)
+
+print("The arguments are: " + str(sys.argv))
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("--workspace_name",
+                        help="Name of the AzureML workspace", type=str, required=True)
+
+parser.add_argument("--resource_group",
+                        help="Resource group of the AzureML workspace", type=str, required=True)
+
+parser.add_argument("--subscription_id",
+                        help="Subscription of the AzureML workspace", type=str, required=True)
+
+parser.add_argument("--gpu_cluster_name",
+                        help="GPU cluster for the script to run on", type=str, required=True)
+
+parser.add_argument("--min_cluster_node",
+                        help="Min number of node for new cluster", type=str, required=True)
+
+parser.add_argument("--max_cluster_node",
+                        help="Max number of node for new cluster", type=str, required=True)
+
+parser.add_argument("--hf_model",
+                        help="Huggingface models to run. One of 'bert-large', 'distilbert-base', 'gpt2', 'bart-large', 't5-large'", type=str, required=True)
+
+parser.add_argument("--run_config",
+                        help="One of pt-fp16, ort, ds_s0, ds_s0_ort, ds_s1, ds_s1_ort", type=str, required=True)
```

Comment:
Nit: Use `choices` argument if there is fixed set of options here. It will raise an error if invalid option is provided.

See e.g. [this so question](https://stackoverflow.com/questions/40324356/python-argparse-choices-with-a-default-choice)

Similarly for args.hf_model.